src/main/java/com/booleanuk/core/Exercise.java

```diff
@@ -91,6 +129,15 @@ public ArrayList<Integer> getFavouriteNumbers() {
          The method must return a boolean that indicates whether the second parameter exists in the provided list
      */
 
+    public boolean containsIngredient(ArrayList<String> ingredients, String ingredient) {
+        if (ingredients.contains(ingredient)) {
+            return true;
+        }
+        else {
+            return false;
+        }
+    }
+
```

Comment:
This could be simplified. For example:

```
public boolean containsIngredient(ArrayList<String> ingredients, String ingredient) {
    return ingredients.contains(ingredient);
}
```

Reply:
Would you like me to edit it?

Reply:
Not necessary, unless you want to. Just a heads up.